pkg/apis/abac/v1beta1/doc.go

```diff
@@ -18,7 +18,9 @@ limitations under the License.
 // +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/abac
 // +k8s:openapi-gen=true
 // +k8s:defaulter-gen=TypeMeta
+// +k8s:openapi-model-gen=true
```

Comment:
Why not simply:

```
+k8s:openapi-model-gen=pkg.apis.abac.v1beta1
```

?

`+groupName` is information about the whoel API (plausibly) but `+modelPackageName` feels like a specific instruction to a tool?  Is thjere any meaning to it outside of this ONE particular thing?